content/docs/design/visual_story@ko.md

```diff
@@ -0,0 +1,26 @@
+---
+$title: 눈이 즐거운 AMP 스토리 제작하기
```

Comment:
Only the title is necessary, the rest of the front matter is inherited from the English file.